specs/dotNetCore1.1.0_should_pass.ps1

```diff
@@ -0,0 +1,10 @@
+Framework "1.1.0"
+
+task default -depends MsBuild
+
+task MsBuild {
+    if ( $IsOSX -OR $IsLinux ) {}
```

Comment:
What's the point of this statement?  Placeholder for future code?  If so, I generally recommending against that.